civicase.php

```diff
@@ -213,6 +213,7 @@ function civicase_civicrm_buildForm($formName, &$form) {
     new CRM_Civicase_Hook_BuildForm_DisplayAllCustomGroupsInCaseForm(),
     new CRM_Civicase_Hook_BuildForm_LinkToCaseSearchByCaseId(),
     new CRM_Civicase_Hook_BuildForm_RemoveExportActionFromReports(),
+    new CRM_Civicase_Hook_BuildForm_PdfFormSubjectRequired(),
```

Comment:
Can we rename this class so it describes what the form does like an adjective similar to the other hooks classes. e.g `MakePdfFormSubjectRequired`